### ISSUE TITLE
UltiSnips private directory is not loaded

### ISSUE THREAD

**Issue by mekanix** [liuchengxu/space-vim#226]
`:UltiSnipsEdit` saves to ~/.space-vim/private/UltiSnips, but I can't run any of the snippets I saved. How to start debugging?

**Comment by liuchengxu** [comment]
Actually, I'm not a heavy snippets user these days. So, chances are great that there are some subtle problems in space-vim, hence maybe it's better not to enable `Layer 'code-snippets'`, but to use `Plug 'SirVer/ultisnips'` in `UserInit()` and add your own config for ultisnips from scratch in `UserConfig()`. If you get it to work finally, you could share us with your config :).

**Comment by liuchengxu** [comment]
You could also try [neosnippet.vim](https://github.com/Shougo/neosnippet.vim), e.g.,

```vim
function! UserInit()
  Plug 'Shougo/neocomplcache'
  Plug 'Shougo/neocomplete'
  Plug 'Shougo/neosnippet'
  Plug 'Shougo/neosnippet-snippets'
endfunction

function! UserConfig()
  imap <C-k>     <Plug>(neosnippet_expand_or_jump)
endfunction
```

**Comment by mekanix** [comment]
https://github.com/SirVer/ultisnips/issues/797#issuecomment-275017642 works for me. Is it OK to change UltiSnips dir to `~/.vim/UltiSnips`?

**Comment by liuchengxu** [comment]
Yup, it's OK.

**Comment by mekanix** [comment]
Closing as it's solved by #227
